src/main/java/org/opentripplanner/openstreetmap/tagmapping/HamburgMapper.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.openstreetmap.tagmapping;
+
+import org.opentripplanner.openstreetmap.model.OSMWithTags;
+
+/**
+ * Modified mapper to allow through traffic for combination access=customers and customers=HVV.
+ *
+ * @see GermanyMapper
+ * @see OsmTagMapper
+ * @see DefaultMapper
```

Comment:
We talked about this in the dev meeting today and have a request: can you add some contact details to the Javadoc for when we have a question about this?

Reply:
Do you have an example where I could see how that should look like?

Reply:
Not really since we came up with this idea yesterday.

A line like this in JavaDoc would be enough:

"Maintained by HBT's OTP team: otp@hbt.de" or "Maintained by Jim Martens: martens@hbt.de"

Reply:
Alright, will add it.